packages/vscode-extension/package.json

```diff
@@ -1141,7 +1141,7 @@
           }
         ],
         "background": {
-          "beginsPattern": "Local tunnel service is started successfully",
+          "beginsPattern": "Forwarding URL https?://.* to https?://.*",
```

Comment:
May bad in previous PR. `beginsPattern` and `endsPattern` cannot be the same, otherwise the debugging will be blocked on this background task.

Reply:
Could it be just .*?

Reply:
No, have to be a pattern after the ngrok starts successfully, otherwise the next task will still be run if starting ngrok fails.